### DISCUSSION TITLE
Sayı dizisi sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#75] [Zeka Sorusu]
5 , 3 , 8 , 5 , 13 , ? , 15 , 5, 29 , 10

Yukarıdaki dizinin bağıntısına göre soru işaretli yere hangi sayı gelmelidir?

**Comment by OmerAkbayrak** [comment]
### 4
**Cevap bence 4, çünkü 5 sayısının yazılışı(beş) 3 harfli ve sonraki sayı 3, sekiz 5 harfli ve sonraki sayı 5, on üç 4 harfli ve sonraki sayı 4 olmalı,  on beş 5 harfli, sonraki sayı 5, yirmi dokuz 10 harfli, sonraki sayı 10**
